tests/unit/FAQTest.php

```diff
@@ -14,30 +26,33 @@ public function testLink()
         $faq = new FAQ();
         $this->assertEquals('', $faq->getLink());
 
+        //TODO Fix the unit tests
```

Comment:
Can remove this comment if fixed

Reply:
no, it's not fixed yet, it's skipping at the moment. we need to fix it later.